packages/gatsby-source-filesystem/src/gatsby-node.js

```diff
@@ -43,7 +43,15 @@ exports.sourceNodes = (
   { actions, getNode, createNodeId, hasNodeChanged, reporter, emitter },
   pluginOptions
 ) => {
-  const { createNode, deleteNode } = actions
```

Comment:
Use `createSchemaCustomization` instead of sourceNodes.

Reply:
If we would use `createSchemaCustomization` here, users would be required to run at least `gatsby@2.12.0` and it will inevitably cause issues for people updating just plugin and not core gatsby package.

IMO, if this is just more correct API to use, but doesn't ultimately change how things work, it's just safer to keep it in `sourceNodes` (at least until we need to do major release for that plugin where we can bump minimal supported Gatsby version without interupting users)